# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../rcs/rcs.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


No comment.  :-)

 - Clifford

--- ./src/conf.sh~	Fri Jun 16 08:19:24 1995
+++ ./src/conf.sh	Fri Jul  3 12:01:16 1998
@@ -73,17 +73,6 @@
 	ech='echo -n' dots='... '
 esac
 
-$ech >&3 "$0: testing permissions $dots"
-rm -f a.d &&
-date >a.d &&
-chmod 0 a.d &&
-{ test -w a.d || cp /dev/null a.d 2>/dev/null; } && {
-	echo >&3 "$n$0: This command should not be run with superuser permissions."
-	exit 1
-}
-echo >&3 OK
-rm -f a.d || exit
-
 $ech >&3 "$0: testing compiler for plausibility $dots"
 echo 'main() { return 0; }' >a.c
 rm -f a.exe a.out || exit
